### PR TITLE
CI: Turn off verbose pytest output

### PR DESCRIPTION
**Pull request by unkcpz**
As mentioned by @agoscinski, @GeigerJ2 , turn off the verbose will make inspect online pytest output log easier.

### CODE REVIEW

**Comment by danielhollas** [comment]
Thanks @unkcpz. Please hold off merging this until #6625 is merged, otherwise the merge conflict is going to be quite annoying.

**Comment by GeigerJ2** [comment]
@unkcpz, I just added a commit that removes the verbosity flag in two more files `nightly.yml` and `release.yml`, just to mirror where @danielhollas removed the flag originally in #6625.

**Review by agoscinski — APPROVED** [review]
Looks good to me

**Pull request merged**
